components/Heading/index.jsx

```diff
@@ -19,7 +19,12 @@ function Heading({ tag, showAnchorIcons, ...props }) {
   if (showAnchorIcons) {
     children.push(
       // eslint-disable-next-line jsx-a11y/anchor-has-content
-      <a key={`heading-anchor-icon-${props.id}`} className="heading-anchor-icon fa fa-anchor" href={`#${props.id}`} />
+      <a
+        key={`heading-anchor-icon-${props.id}`}
+        aria-label={`Skip link for ${props.children[1]}`}
```

Comment:
```suggestion
        aria-label={`Skip link to ${props.children[1]}`}
```

NIT: I kind of like `to` instead of `for` just to make it a little clearer of the functionality.

Reply:
Good catch!